packages/types/src/cart/mutations.ts

```diff
@@ -0,0 +1,159 @@
+export interface UpsertAddressDTO {
```

Comment:
comment: I can see that we have 3 ways of storing the DTO's, in product, everything is in one file, in this one some are in common and others are in mutations, and for the other modules, we have a common directory and it is splitted by entity. should we align on that?

Reply:
Yes, good idea. I personally found one file unmanageable, which is why I decided to separate types for mutations out. If there are no strong arguments against this, I think we should do something along those lines to make the types easier to navigate through and use.

Reply:
I think it is better to have it splitter like you did